src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1356,6 +1356,12 @@
   <data name="ERR_MethGrpToNonDel" xml:space="preserve">
     <value>Cannot convert method group '{0}' to non-delegate type '{1}'. Did you intend to invoke the method?</value>
   </data>
+  <data name="WRN_MethGrpToNonDel" xml:space="preserve">
+    <value>Converting method group '{0}' to non-delegate type '{1}'. Did you intend to invoke the method?</value>
+  </data>
+  <data name="WRN_MethGrpToNonDel_Title" xml:space="preserve">
+    <value>Converting method group to non-delegate type</value>
+  </data>
```

Comment:
nit: Doesn't feel like a complete statement like most other warning titles.

Reply:
This is the normal style our warning titles have. Consider `WRN_AmbiguousXMLReference_Title` just above this, for example.

Reply:
@333fred Looks like both styles are used.

Reply:
Will leave as is.